GitHubCore.ps1

```diff
@@ -272,7 +280,7 @@ function Invoke-GHRestMethod
                     }
                 }
 
-                $null = Start-Job -Name $jobName -ScriptBlock $scriptBlock -Arg @($url, $Method, $headers, $Body, $ValidBodyContainingRequestMethods, (Get-GitHubConfiguration -Name WebRequestTimeoutSec), $PSScriptRoot)
+                $null = Start-Job -Name $jobName -ScriptBlock $scriptBlock -Arg @($url, $Method, $headers, $Body, $ValidBodyContainingRequestMethods, (Get-GitHubConfiguration -Name WebRequestTimeoutSec), (Get-GitHubConfiguration -Name LogRequestBody), $PSScriptRoot)
```

Comment:
Is it time yet to start splitting these arguments across separate lines?
```powershell
$null = Start-Job -Name $jobName -ScriptBlock $scriptBlock -Arg @(
    $url,
    $Method
    ...
)
```

Reply:
At this point, I'd agree. Will fix.